packages/sublime_merge.rb

```diff
@@ -0,0 +1,41 @@
+require 'package'
+
+class Sublime_merge < Package
+  description 'A snappy UI, three-way merge tool,side-by-side diffs, syntax highlighting, and more.'
+  homepage 'https://www.sublimemerge.com/'
+  version 'Build 1065'
+  case ARCH
+  when 'x86_64'		  
+	source_url 'https://download.sublimetext.com/sublime_merge_build_1065_x64.tar.xz'
+	source_sha256 'ebe9029ea71543a101dcfd09c5540759759c1e33fea31d4b98256803b7ceaf97'
+  else
+    puts 'Merge is not supported on your system yet.'
```

Comment:
Probably should just remove this message, otherwise, it shows up when you `crew search merge` on arm or i686.

Reply:
Do you want me to remove the entire conditional or just the else statement?

Reply:
Just the else and message lines.